libbeat/beat/beat.go

```diff
@@ -155,6 +156,10 @@ func (b *Beat) launch(bt Creator) error {
 		return err
 	}
 
+	if err := plugin.Initialize(); err != nil {
```

Comment:
Plugins are loadable from CLI only using the `-plugin` setting. No config file support yet, as feature is very experimental.

Only if plugin support is available, the flag will be available. The Initialize function is implemented in [cli.go](https://github.com/elastic/beats/pull/3217/files#diff-9ba51f06e4b345b7121a0f8cd521c46bR33). In case no plugin support is available [cli_stub.go](https://github.com/elastic/beats/pull/3217/files#diff-87fbcf05836a787e64b638f81e89f252R5) will be used (empty function).

This behavior close enough?